pyuvdata/uvcal/uvcal.py

```diff
@@ -561,6 +563,45 @@ def __init__(self):
 
         super(UVCal, self).__init__()
 
+    @staticmethod
+    def new(**kwargs):
+        """
+        Create a new UVCal object.
+
+        All parameters are passed through to the
+        :func:`~pyuvdata.uvcal.initializers.new_uvcal` function.
+        """
+        return initializers.new_uvcal(**kwargs)
+
+    @classmethod
+    def from_uvdata(cls, uvd: UVData, **kwargs):
+        """
+        Create a new UVCal object from a UVData object.
+
+        All parameters are passed through to the
+        :func:`~pyuvdata.uvcal.initializers.new_uvdata` function, but those that can
```

Comment:
Shouldn't this point to the `new_uvcal` function not the `new_uvdata` function? I get why you don't want to copy/paste docstrings here, my first choice would be to get all the parameters in like you did on uvdata, but if that's not possible maybe we can instead point to the `new` method (which will have all the options) so that we are just sending users to another method on the same object?

Reply:
And are we really going to have both a `from_uvdata` and an `initialize_from_uvdata` class method on UVCal long term? I'd like us to go with one or the other. If this is the one we're going with, the other one should have a deprecation warning if it is called.

Reply:
I don't think I meant to leave this method in... I've deleted it. I certainly don't mind renaming `initialize_from_uvdata` to `from_uvdata`, but we don't have to, and maybe for simplicity we just leave it as-is.